code/Test_definitions/population-density-data.feature

```diff
@@ -0,0 +1,277 @@
+        And the header "x-correlator" is set to a UUID value
+        And the request body is set by default to a request body compliant with the schema
+
+    # Happy path scenarios
+
+    @population_density_data_01_supported_area_success_scenario
+    Scenario: Validate success response for a supported area request
+        Given the request body property "$.area" is set to a valid testing area within supported regions
+        And the request body property "$.startDate" is set to a valid testing future date
+        And the request body property "$.endDate" is set to a valid testing future date later than body property "$.startDate"
+        When the request "retrievePopulationDensity" is sent
+        Then the response status code is 200
+        And the response header "Content-Type" is "application/json"
+        And the response header "x-correlator" has same value as the request header "x-correlator"
+        And the response body complies with the OAS schema at "/components/schemas/PopulationDensityResponse"
+        And the response property "$.status" value is "SUPPORTED_AREA"
+        And the response property "$.timedPopulationDensityData[*].startTime" is equal to or later than request body property "$.startDate"
+        And the response property "$.timedPopulationDensityData[*].endTime" is equal to or earlier than request body property "$.endDate"
+        And the response property "$.timedPopulationDensityData[*].cellPopulationDensityData[*].populationDensityData.dataType.datatype" is equal to "LOW_DENSITY" or "DENSITY_ESTIMATION"
+
+    @population_density_data_02_partial_area_success_scenario
+    Scenario: Validate success response for  a partial supported area request
+        Given the request body property "$.area" is set to a valid testing area partially within supported regions
+        And the request body property "$.startDate" is set to a valid testing future date
+        And the request body property "$.endDate" is set to a valid testing future date later than body property "$.startDate"
+        When the request "retrievePopulationDensity" is sent
+        Then the response status code is 200
+        And the response header "Content-Type" is "application/json"
+        And the response header "x-correlator" has same value as the request header "x-correlator"
+        And the response body complies with the OAS schema at "/components/schemas/PopulationDensityResponse"
+        And the response property "$.status" value is "PART_OF_AREA_NOT_SUPPORTED"
+        And the response property "$.timedPopulationDensityData[*].startTime" is equal to or later than request body property "$.startDate"
+        And the response property "$.timedPopulationDensityData[*].endTime" is equal to or earlier than request body property "$.startDate"
+        And there is at least one item in response property "$.timedPopulationDensityData[*].cellPopulationDensityData[*].datatype" equal to "NO_DATA"
```

Comment:
and add:
`And there is at least one item in response property "$.timedPopulationDensityData[*].cellPopulationDensityData[*].datatype" equal to "LOW_DENSITY" or "DENSITY_ESTIMATION"`

Reply:
to be solved

Reply:
For me yes :) line 61

Reply:
Thanks ludovic, solved then.